src/strings/PowerFxResources.en-US.resx

```diff
@@ -6436,4 +6436,16 @@
   <data name="ErrNamedFormula_AlreadyDefined" xml:space="preserve">
     <value>NamedFormula '{0}' already exists.</value>
   </data>
+  <data name="ErrNeedValidVariableName_Arg" xml:space="preserve">
+    <value>The first argument of '{0}' should be a valid variable name, and cannot conflict with any existing control, screen, collection, or data source names. Found type '{1}'</value>
```

Comment:
>'{1}' [](http://example.com/codeflow?start=172&length=5)

This will not be a type but name that is used by the end-user. I think Found type part is not needed.

I would change the message it

Invalid variable name {1} used as a first argument of function {0}. It should be a valid variable name, and cannot conflict with any existing control, screen, collection, or data source names.

Reply:
I can update  - fwiw, I just copied from Canvas apps.

Reply:
Ok, then I'll leave as is to stay consistent with Canvas.